.env

```diff
@@ -0,0 +1,5 @@
+PGHOST='localhost'
+PGUSER=process.env.USER
+PGDATABASE=process.env.USERnpm
```

Comment:
why do you get environment variables in env file? Env file should set env variables